dolphinscheduler-ui/src/views/projects/workflow/definition/components/use-form.ts

```diff
@@ -66,7 +66,7 @@ export const useForm = () => {
       tenantCode: 'default',
       environmentCode: null,
       startParams: null,
-      expectedParallelismNumber: '',
+      expectedParallelismNumber: '2',
```

Comment:
Why the default value is 2 rather than 3? If you want to set a default value, I would prefer 1 or a default empty value.

Reply:
I understand that if parallel mode is selected, it should be greater than 1, otherwise just select serial mode. Having default values ​​will make the user experience more friendly and allow users to do as few operations as possible.

Reply:
If my habit is to use maximum parallelism every time, then this change will not be friendly to me.

Reply:
> If my habit is to use maximum parallelism every time, then this change will not be friendly to me.

You could also use the maximum parallelism 100. But limit-free will be risky to the system.

Reply:
If this will bring risky, there will exist a lot of risky in the system. We need to limit the scheduler number, limit the task number in one workflow, limit the size (parallelism in each backfill * backfill request number)...... The main thing is that why this will bring risky to the system, if this will bring risky to system we should fix this in a correct way, rather than simple limit the input form, can this restriction fix this problem? From my point, this restriction just make it more inconvinent for me to use, but it does not solve the problem caused by the high stress.

Reply:
Not using the maximum parallelism has another meaning, ensuring the execution of other normal scheduled tasks. If the user directly uses the maximum degree of parallelism, the normally scheduled tasks will not be executed normally. I understand that the user needs to know whether his complement tasks during this period will affect other tasks before selecting the maximum degree of parallelism.

Reply:
OK, this make sense to me, the maximum paramllelism does really will affect the current running scheduler workflow. We should direct users to try not to use such large parallel reads unless there is some special reason.